Update Node.js 2.X SDK to Azure Monitor Distro Migration Guide (#123553)
Azure/azure-content: articles/azure-monitor/app/opentelemetry-nodejs-migrate.md

    Remove all Application Insights instrumentation from your code. Delete any sections where the Application Insights client is initialized, modified, or called.

4. Enable Application Insights with the Azure Monitor OpenTelemetry Distro.

>     > *Important:* `useAzureMonitor` must be called *before* you import anything else. There may be resulting telemetry loss if other libraries are imported first.

Reviewer: ```suggestion
    > [!IMPORTANT] 
    > *Before* you import anything else, `useAzureMonitor` must be called. There might be telemetry loss if other libraries are imported first.
```
Fixing formatting and making minor edits for Microsoft style.
